src/sinks/util/encoding.rs

```diff
@@ -375,4 +381,120 @@ mod tests {
         assert_eq!(String::from_utf8(writer).unwrap(), r"value");
         assert_eq!(CountByteSize(1, input_json_size), json_size.size().unwrap());
     }
+
+    #[test]
+    fn test_encode_batch_protobuf_single() {
+        let message_raw = std::fs::read("tests/data/protobuf/test_proto.pb").unwrap();
```

Comment:
Nit:

```suggestion
        let message_raw = std::fs::read(test_data_dir().join("test_proto.pb").unwrap();
```

Same for the paths below.

Reply:
thanks, have made the change in [5c9fd29](https://github.com/vectordotdev/vector/pull/22536/commits/5c9fd29b2f221f89c4185e54ea439d0379c7c887)